assets/js/modules/reader-revenue-manager/components/dashboard/ReaderRevenueManagerSetupCTABanner.js

```diff
@@ -37,268 +35,164 @@ import { __ } from '@wordpress/i18n';
-				dismissCount < 1 ? 2 * WEEK_IN_SECONDS : 0;
-
-			showTooltip();
-
-			dismissPrompt( READER_REVENUE_MANAGER_SETUP_BANNER_DISMISSED_KEY, {
-				expiresInSeconds: expirationInSeconds,
-			} );
-		} );
-	}, [ dismissCount, dismissPrompt, showTooltip, viewContext ] );
-
-	const readerRevenueManagerDocumentationURL =
-		'https://readerrevenue.withgoogle.com';
-
-	const canActivateRRMModule = useSelect( ( select ) =>
-		select( CORE_MODULES ).canActivateModule(
-			READER_REVENUE_MANAGER_MODULE_SLUG
-		)
-	);
-
-	const showBanner =
-		isDismissed === false &&
-		canActivateRRMModule &&
-		dismissedPromptsLoaded === true;
+	const hideCTABanner = isCTADismissed || ! dismissedPromptsLoaded;
 
 	useEffect( () => {
-		if ( showBanner ) {
-			trackEvent(
-				`${ viewContext }_rrm-setup-notification`,
-				'view_notification'
-			);
-
+		if ( ! hideCTABanner ) {
 			triggerSurvey( 'view_reader_revenue_manager_cta' );
 		}
-	}, [ showBanner, triggerSurvey, viewContext ] );
+	}, [ hideCTABanner, triggerSurvey ] );
 
 	if ( isTooltipVisible ) {
 		return (
 			<Fragment>
-				<WidgetNull />
 				<AdminMenuTooltip
 					title=""
 					content={ __(
 						'You can always enable Reader Revenue Manager from Settings later',
 						'google-site-kit'
 					) }
 					dismissLabel={ __( 'Got it', 'google-site-kit' ) }
-					tooltipStateKey={
-						READER_REVENUE_MANAGER_SETUP_BANNER_DISMISSED_KEY
-					}
+					tooltipStateKey={ id }
 				/>
 			</Fragment>
 		);
 	}
 
-	if ( ! showBanner ) {
-		return <WidgetNull />;
+	// TODO Remove this hack
+	// We "incorrectly" pass true to the `skipHidingFromQueue` option when dismissing this banner.
+	// This is because we don't want the component removed from the DOM as we have to still render
+	// the `AdminMenuTooltip` in this component. This means that we have to rely on manually
+	// checking for the dismissal state here.
+	if ( hideCTABanner ) {
+		return null;
 	}
 
+	const breakpointSVGMap = {
+		[ BREAKPOINT_SMALL ]: SetupMobileSVG,
+		[ BREAKPOINT_TABLET ]: SetupTabletSVG,
+	};
+
 	return (
-		<div className="googlesitekit-widget-context">
-			<Grid className="googlesitekit-widget-area">
-				<Row>
-					<Cell size={ 12 }>
-						<Widget
-							noPadding
-							className="googlesitekit-setup-cta-banner googlesitekit-reader-revenue-manager-setup-cta-widget"
-						>
-							<Grid collapsed>
-								<Row>
-									<Cell
-										smSize={ 12 }
-										mdSize={ 8 }
-										lgSize={ 6 }
-										className="googlesitekit-setup-cta-banner__primary-cell"
-									>
-										<h4 className="googlesitekit-setup-cta-banner__title">
-											{ __(
-												'Grow your revenue and deepen reader engagement',
+		<Notification>
+			<NotificationWithSVG
+				id={ id }
+				title={ __(
+					'Grow your revenue and deepen reader engagement',
+					'google-site-kit'
+				) }
+				description={
+					<div className="googlesitekit-setup-cta-banner__description">
+						<p>
+							{ ' ' }
```

Comment:
Is this intentional? Not sure why we'd need it.

Reply:
Yup, sorry, this was probably added by ESLint's autosave when doing some string copy pasting..